src/main/java/sirius/web/data/ExcelExport.java

```diff
@@ -399,6 +404,17 @@ public ExcelExport addRow(Object... row) {
      * @return the export itself for fluent method calls
      */
     public ExcelExport addRowAsList(Collection<?> row) {
+        if (rows > MAX_NUM_ROWS) {
```

Comment:
shouldn't we zero this once the sheet is changed or is this a global limit?

Reply:
we do in the `createSheet`-method